packages/sdkman.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Install sdkman with support for Codeship dependency caching, http://sdkman.io/
+#
+# Include in your builds via
+# source /dev/stdin <<< "$(curl -sSL https://raw.githubusercontent.com/codeship/scripts/master/packages/sdkman.sh)"
+
+set -e
```

Comment:
If this script is sourced, we can't have `set -e` in it.

Longer explanation, `set -e` will stay active and cause the first failing command to kill the build with a _System Error_. Even longer explanation is available at https://community.codeship.com/t/npm-test-a-b-c-where-b-throws-but-result-shows-ok/61/3